ui/src/plugins/epl-ets/sample_data.json

```diff
@@ -0,0 +1,8671 @@
+{
```

Comment:
You can delete this file from since we're not using it. You can delete the same file from `epl-vizceral` too.